tests/components/sensor/test_fail2ban.py

```diff
@@ -99,101 +98,122 @@ def test_multi_jails(self):
 
     def test_single_ban(self):
         """Test that log is parsed correctly for single ban."""
-        log_parser = BanLogParser(timedelta(seconds=-1), '/tmp')
+        log_parser = BanLogParser('/tmp')
         sensor = BanSensor('fail2ban', 'jail_one', log_parser)
-        assert sensor.name == 'fail2ban jail_one'
+        self.assertEqual(sensor.name, 'fail2ban jail_one')
         mock_fh = MockOpen(read_data=fake_log('single_ban'))
         with patch('homeassistant.components.sensor.fail2ban.open', mock_fh,
                    create=True):
             sensor.update()
 
-        assert sensor.state == '111.111.111.111'
-        assert \
-            sensor.state_attributes[STATE_CURRENT_BANS] == ['111.111.111.111']
-        assert \
-            sensor.state_attributes[STATE_ALL_BANS] == ['111.111.111.111']
+        self.assertEqual(sensor.state, '111.111.111.111')
```

Comment:
Don't change things that we don't need to change in the tests.

Reply:
Fixed in latest commit